sync/class.jetpack-sync-actions.php

```diff
@@ -135,11 +136,19 @@ static function send_data( $data, $codec_name, $sent_timestamp, $queue_id ) {
 		return $rpc->getResponse();
 	}
 
+	static function get_initial_sync_user_config() {
+		$user_query = new WP_User_Query( array( 'who' => 'authors', 'fields' => 'ID' ) );
```

Comment:
Could we set a limit to WP_User_Query to be MAX_INITIAL_SYNC_USER + 1. 
This way the query wouldn't have to do as much work.